struct/interfaces.ts

```diff
@@ -0,0 +1,29 @@
+export interface DefaultOptions {
```

Comment:
```suugestion
export const 
```

Reply:
Interface is defining the type

Reply:
Yeah but... Isn't that an obejct

Reply:
const is just an object interface is defining the type and if you look in the manager it is used as an type

Reply:
Ah alright